openupgradelib/openupgrade.py

```diff
@@ -227,8 +227,8 @@ def copy_columns(cr, column_spec):
     values. Tuples consist of (old_name, new_name, type). Use None for \
     new_name to trigger a conversion of old_name using get_legacy_name() \
     Use None for type to use type of old field.
-    Make sure to quote properly, if your column name coincides with a
-    SQL directive. eg. '"column"'
+        Make sure to quote properly, if your column name coincides with a
+        SQL directive. eg. '"column"'
```

Comment:
It would be more readable if you change the above `\`s for indents too if you don't mind.